fixtures/dummy/config.go

```diff
@@ -6,43 +6,55 @@ import (
 )
 
 var EKSCluster = models.ConfigItem{
-	ID: uuid.New(),
+	ID:         uuid.New(),
+	ConfigType: "kubernetesCluster",
 }
 
 var KubernetesCluster = models.ConfigItem{
-	ID: uuid.New(),
+	ID:         uuid.New(),
+	ConfigType: "kubernetesCluster",
 }
 
 var KubernetesNodeA = models.ConfigItem{
-	ID: uuid.New(),
+	ID:           uuid.New(),
+	ConfigType:   "kubernetesNode",
+	CostTotal30d: 1,
 }
 
 var KubernetesNodeB = models.ConfigItem{
-	ID: uuid.New(),
+	ID:           uuid.New(),
+	ConfigType:   "kubernetesNode",
```

Comment:
Make these Capitalized Constants in models/config_item.go

We will eventually use them directly in conifg-db